truss/api/__init__.py

```diff
@@ -4,6 +4,26 @@
 from truss.api import definitions
 from truss.remote.baseten.service import BasetenService
 from truss.remote.remote_factory import RemoteFactory
+from truss.remote.truss_remote import RemoteConfig
+
+
+def login(api_key: str):
+    """
+    Logs user into Baseten account. Persists information to ~/.trussrc file,
+    so only needs to be invoked once.
+    Args:
+        api_key: Baseten API Key
+    """
+    remote_url = "https://app.baseten.co"
```

Comment:
should we allow this to be an optional kwarg?

Reply:
Since this `login` method is part of the public truss API, I don't think we should necessarily expose this here, changing the remote is a fairly advanced feature that only baseten people really use, and for them it's ok to modify ~/.trussrc directly

Reply:
I suppose I was thinking of an on-prem deployment case, but if that's handled by the main CP url, then it should be fine.

Reply:
I see -- lets tackle that when we get to that, I know there are going to be a couple different URLs in play there